operation/fhir-operation-bulkdata/src/main/resources/OperationDefinition-patient-export.json

```diff
@@ -83,14 +83,6 @@
       "max": "1",
       "documentation": "A string of comma-delimited FHIR resource types. Only resources of the specified resource types(s) SHOULD be included in the response. If this parameter is omitted, the server SHOULD return all supported resources within the scope of the client authorization. For non-system-level requests, the Patient Compartment SHOULD be used as a point of reference for recommended resources to be returned as well as other resources outside of the patient compartment that are helpful in interpreting the patient data such as Organization and Practitioner. Resource references MAY be relative URIs with the format <resource type>/<id>, or absolute URIs with the same structure rooted in the base URI for the server from which the export was performed. References will be resolved looking for a resource with the specified type and id within the file set. Note: Implementations MAY limit the resources returned to specific subsets of FHIR, such as those defined in the Argonaut Implementation Guide.",
       "type": "string"
-    },
-    {
-      "name": "_typeFilter",
-      "use": "in",
-      "min": 0,
-      "max": "1",
-      "documentation": "A string of comma-delimited type filters.",
-      "type": "string"
```

Comment:
i'm disappointed the removed these from the OperationDefinition.  I think we want to continue supporting these...we might want to add it back in :-(

Reply:
Adding back in.